woocommerce/wc-admin
# Update E2E config to support screenshots on test failure

Make use of the `useE2EJestConfig` function so it correctly uses the `e2e-environment` libraries `setupFilesAfterEnv` files.
This also uploads the screenshots if the exist (only on test failure) as artifacts in Github actions.

### Detailed test instructions:

- Make sure that e2e github action passes correctly, here is a sample of a [past run that failed](https://github.com/woocommerce/woocommerce-admin/actions/runs/778144422) with a generated artifact.
- Modify one of the e2e tests to make sure it fails (locally) 
- Run the tests using `npx wc-e2e docker:up && WC_E2E_SCREENSHOTS=1 npx wc-e2e test:e2e`
- Check the `tests/e2e/screenshots` folder, it should contain a screenshot for the failed test.
